src/ace.F90

```diff
@@ -482,6 +482,15 @@ subroutine read_esz(nuc, data_0K)
       ! Continue reading elastic scattering and heating
       nuc % elastic = get_real(NE)
 
+      ! Determine if minimum/maximum energy for this nuclide is greater/less
+      ! than the previous
+      energy_min_neutron = max(energy_min_neutron, nuc%energy(1))
+      energy_max_neutron = min(energy_max_neutron, nuc%energy(NE))
+      if (nuc%energy(NE) < 20.0_8) then
```

Comment:
Ok, I've replaced the warning with a normal message giving the maximum neutron transport energy and corresponding nuclide. The calculated minimum is not actually used to truncate the range, i.e. neutrons are allowed below 1e-5 eV which is consistent with other MC codes, so no message is displayed for the minimum.